types/esrecurse/index.d.ts

```diff
@@ -36,12 +36,6 @@ declare class Visitor {
      * @param node The AST node whose children to visit.
      */
     visitChildren(node: any): void;
-
```

Comment:
Is this not a breaking change?

Reply:
Actually, this is added by me in a previous PR some weeks ago. Back then I thought it may be a good idea to "polyfill" method given how the example works, but now seeing that, I think this may not be the way to do, so the removal of this part is kinda trying to make things right.

Reply:
Which PR?

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72705/files